content/develop/start/truffle.md

```diff
@@ -185,16 +184,15 @@ function safeTransferFrom(
 
 Participants can transfer the token to the minter after receiving the vaccine.
 As shown below, a participant can only send the NFT token if the receiver for
-this token is the minter (`0x6A33382de9f73B846878a57500d055B981229ac4`).
+this token is the minter (`accounts[0]`).
 
 ```bash
-truffle(aurora)> await cvt.ownerOf(1) // TokenID 1
-'0x2531a4D108619a20ACeE88C4354a50e9aC48ecfe'
-truffle(aurora)> await cvt.minter()
-'0x6A33382de9f73B846878a57500d055B981229ac4'
-truffle(aurora)> await cvt.safeTransferFrom('0x2531a4D108619a20ACeE88C4354a50e9aC48ecfe', '0x6A33382de9f73B846878a57500d055B981229ac4', 1 , {from: accounts[1]})
-truffle(aurora)> await cvt.ownerOf(1) // TokenID 1
-'0x6A33382de9f73B846878a57500d055B981229ac4'
+truffle(aurora)> const tokenID = 1
+truffle(aurora)> await cvt.ownerOf(tokenID) == accounts[1]
+true
+truffle(aurora)> await cvt.safeTransferFrom(accounts[1], accounts[0], tokenID, {from: accounts[1]})
```

Comment:
I get a nonce error at this point in the tutorial. It is easy to work around by either explicitly setting the nonce in the last argument (value taken from `web3.eth.getTransactionCount(accounts[1])`), or by exiting the console and going back in.

I think this happens because truffle doesn't update the nonce after the previous transaction since it threw the `Invalid Transfer` error. I'm not sure if there is anything we can do about this. Is it normally for Truffle to lose track of the nonce when transactions fail?

Reply:
>> I get a nonce error at this point in the tutorial. It is easy to work around by either explicitly setting the nonce in the last argument (value taken from web3.eth.getTransactionCount(accounts[1])), or by exiting the console and going back in.

Not sure about the `nonce error` but it looks like you need to share the nonce between accounts through the HDWallet.
Some times if fails but as a workaround, you can export the `MNEMONIC` again and re-run your truffle script.


>> I think this happens because truffle doesn't update the nonce after the previous transaction since it threw the Invalid Transfer error. I'm not sure if there is anything we can do about this. Is it normally for Truffle to lose track of the nonce when transactions fail?

I've no clue about this.